Application/EdFi.Ods.Features/Controllers/OpenApiMetadataController.cs

```diff
@@ -6,6 +6,7 @@
 using System;
 using System.Linq;
 using System.Net;
+using System.Net.Http.Json;
```

Comment:
This using statement is newly added but does not seem to be necessary:
using System.Net.Http.Json;